brothers.html

```diff
@@ -8,7 +8,7 @@
     content="Delta Sigma Phi Fraternity | Zeta Lambda Chapter | Rose-Hulman Institute of Technology">
 
   <link rel="shortcut icon" href="ico/favicon.ico">
-  <link type="text/css" rel="stylesheet" href="css/materialize.css" media="screen,projection" />
+  <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection" />
```

Comment:
We're yeeting `css/materialize.min.css`, does it get auto-generated?

Reply:
It seems like its not getting yeeted, but rather condensed all into line six for viewing sake. Is this accurate @DavidPurdy1 ?

Reply:
oh you right. I assumed 7500 lines of deletion was a full yeet.